### PR TITLE
Tx to user

### PR DESCRIPTION
**Pull request by tharvik**
* add send_asset agent
* show dialog to send asset

`cargo deny` is still failing due to recent RUSTSEC

### CODE REVIEW

File: web/src/config.rs [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
 const NAME_SERVICE_URI: &str = "https://factory.c4dt.org/incubator/at2/demo/ns";
+const NETWORK_URI: &str = "https://factory.c4dt.org/incubator/at2/demo/leader";
 
 pub struct Config {
     name_service: Uri,
+    network: Uri,
 }
 
 impl Config {
     pub fn parse() -> Result<Self, http::uri::InvalidUri> {
```
Comment (ineiti) [inline web/src/config.rs:14]: Do I understand correctly that this is a stub and should later be replaced by a real config-file? If yes, please add a `TODO` comment and say so.

File: web/src/config.rs [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
             name_service: NAME_SERVICE_URI.parse()?,
+            network: NETWORK_URI.parse()?,
         })
     }
 
     pub fn name_service(&self) -> &Uri {
         &self.name_service
     }
+
+    pub fn network(&self) -> &Uri {
```
Comment (ineiti) [inline web/src/config.rs:25]: Another long discussion: getters for simple fields that are not cloned, nor modified...

File: web/src/pages/your_account.rs [hunk 3]
```diff
@@ -1,49 +1,140 @@
...
+struct MissingField;
+
+impl fmt::Display for MissingField {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str("missing field")
+    }
+}
+impl std::error::Error for MissingField {}
+
+// TODO split
```
Comment (ineiti) [inline web/src/pages/your_account.rs]: are these TODOs for later?

**Review by ineiti — COMMENTED** [review]
Can I haz some context? Some comments in the new files, at least 2-3 phrases of what it's doing?

File: web/src/config.rs (hunk 2 above)
Comment (tharvik) [reply web/src/config.rs:25]: It's a way to force the object to be immutable, as there is no way to change an already existing `Config` (and zero cost in Rust).

File: web/src/config.rs (hunk 1 above)
Comment (tharvik) [reply web/src/config.rs:14]: Right, it's temporary, adding a `TODO` line.

File: web/src/pages/your_account.rs (hunk 3 above)
Comment (tharvik) [reply web/src/pages/your_account.rs]: Good catch, it was to be handled before the PR, fixing it now :)

**Review by ineiti — APPROVED** [review]

**Pull request merged**